src/backdraft.js

```diff
@@ -3,6 +3,13 @@
   var Backdraft = {};
   Backdraft.Utils = {};
 
+  // use squiggly braces for underscore templating so we don't conflict with ruby templating
+  _.templateSettings = {
+    evaluate    : /{{([\s\S]+?)}}/g,
+    interpolate : /{{=([\s\S]+?)}}/g,
+    escape      : /{{-([\s\S]+?)}}/g
+  }
+
```

Comment:
Discussed with @nburwell 
This will have to do for now.  We can consider going back to the defaults in the future, but it will require a STORY on invoca/web/